docs/ABI/Mangling.rst

```diff
@@ -467,7 +467,6 @@ Types
 
   assoc-type-list ::= assoc-type-name '_' assoc-type-name*
 
-  archetype ::= context 'Qq' INDEX           // archetype+context (DWARF only)
   archetype ::= associated-type
```

Comment:
Can you remove this line too?

Reply:
which one?

Reply:
archetype ::== associated-type

There’s no archetype production any longer